tests/test_api.py

```diff
@@ -294,6 +294,14 @@ def test_describe_processes(config, api_):
     assert len(process['outputTransmission']) == 1
     assert len(process['jobControlOptions']) == 1
 
+    api_.config['processes'] = {}
+
+    req_headers = make_req_headers()
+    rsp_headers, code, response = api_.describe_processes(
+        req_headers, {}, 'foo')
+    processes = json.loads(response)
+    assert len(processes['processes']) == 0
+
```

Comment:
have you seen my suggestion for additional test? The essence of this issue is to allow `processes` to be left out of config (right?) but that case is never tested. Could even be done with a loop:

```
empty_vals = [None, {}]
for empty_val in empty_vals:
    api_.config['processes'] = empty_val

    req_headers = make_req_headers()
    rsp_headers, code, response = api_.describe_processes(
        req_headers, {}, 'foo')
    processes = json.loads(response)
    assert len(processes['processes']) == 0
```